### ISSUE TITLE
Editors can not use ordered list inside un-ordered list (vice versa)

### ISSUE THREAD

**Issue by qh-huaweichen** [Qld-Health-Online-Team/design-system#100]
# TL;DR

Editors cannot use `<ul>` and `<ol>` nest each other. But this is a very common requirement.

# TS;WM

As an editor, I want to have 
![image](https://github.com/Qld-Health-Online-Team/design-system/assets/130127944/80c84bfe-0d49-4839-a9e6-187f600730f7)

But the design system handled the WYSIWYG as:
![image](https://github.com/Qld-Health-Online-Team/design-system/assets/130127944/99ac7ec3-22b3-43a3-bfbd-26bf4c079ee7)

#### We don't want to hack our DOM element with extra unnecessary classes.

Feels like the related code is here:
https://github.com/Qld-Health-Online-Team/design-system/blob/master/src/components/_global/css/body/component.scss#L395...#L422

**Comment by manjeetk** [comment]
@qh-huaweichen 
Design System encourages a maximum of two levels of nesting for multilevel lists, following the Australian Government Style Manual https://www.stylemanual.gov.au/structuring-content/lists#avoid_using_a_multilevel_list 
https://www.design-system.health.qld.gov.au/styles/typography#section__lists
